From 636e846395106af95d73744cf0df7bdb9f304c03 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Thu, 13 Oct 2016 17:30:58 -0700
Subject: [PATCH] MANTA-2998 marlin's dev/package.json is broken by latest
 node-sdc-clients Reviewed by: David Pacheco <dap@joyent.com>

---
 common/lib/ufds.js | 2 +-
 dev/package.json   | 2 +-
 dev/test/common.js | 4 ++--
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/common/lib/ufds.js b/common/lib/ufds.js
index d4ad1d2..7dfac94 100644
--- a/common/lib/ufds.js
+++ b/common/lib/ufds.js
@@ -41,7 +41,7 @@ exports.ufdsMakeAccounts = ufdsMakeAccounts;
  *     			if we develop a better approach to waiting on
  *     			replication), this dependency could be removed.
  *
- *     ufds		UFDS client object (from node-sdc-clients.git)
+ *     ufds		UFDS client object (from node-ufds.git)
  *
  *     config		A configuration object, which has this general form:
  *
diff --git a/dev/package.json b/dev/package.json
index 83f1d7c..77f8469 100644
--- a/dev/package.json
+++ b/dev/package.json
@@ -40,7 +40,7 @@
 		"restify": "2.3.5",
 		"retry": "0.6.0",
 		"semver": "1.1.4",
-		"sdc-clients": "git+https://github.com/joyent/node-sdc-clients.git#master",
+		"ufds": "1.2.0",
 		"spawn-async": "0.0.1",
 		"statvfs": "2.0.0",
 		"strsplit": "1.0.0",
diff --git a/dev/test/common.js b/dev/test/common.js
index cdf6e77..d1f7d60 100644
--- a/dev/test/common.js
+++ b/dev/test/common.js
@@ -30,7 +30,7 @@ var mod_verror = require('verror');
 var mod_mahi = require('mahi');
 var mod_manta = require('manta');
 var mod_marlin = require('../lib/marlin');
-var mod_sdc = require('sdc-clients');
+var UFDS = require('ufds');
 var mod_maufds = require('../lib/ufds');
 
 var VError = mod_verror.VError;
@@ -97,7 +97,7 @@ function setup(callback)
 
 	barrier.start('ufds');
 	log.info({ 'url': ufds_url }, 'connecting to ufds');
-	ufds_client = new mod_sdc.UFDS({
+	ufds_client = new UFDS({
 	    'url': ufds_url,
 	    'bindDN': 'cn=root',
 	    'bindPassword': 'secret'
-- 
2.21.0

